.gitignore

```diff
@@ -24,3 +24,6 @@ pod.yaml
 # Things running grandmatriarch locally might create
 cookies
 secret.yaml
+
+# logexported binary
```

Comment:
Do we need this? I won't be able to approve changes in this file

Reply:
No, I'll move this to a separate PR.